src/xdsl/diagnostic/xdsl-diagnostic.v6.service.js

```diff
@@ -6,6 +6,7 @@ angular.module("ovh-api-services").service("OvhApiXdslDiagnosticV6", function ($
     var diagnostic = $resource(route, {
         xdslId: "@xdslId"
     }, {
+        get: { method: "GET" },
```

Comment:
Btw I think this is not needed.

ref: https://github.com/angular/angular.js/blob/master/src/ngResource/resource.js#L214-L224

Reply:
Oh, nice!